mcserver/views.py

```diff
@@ -861,24 +1234,23 @@ class TrialViewSet(viewsets.ModelViewSet):
     queryset = Trial.objects.all().order_by("created_at")
     serializer_class = TrialSerializer
 
-    permission_classes = [IsPublic | ((IsOwner | IsAdmin | IsBackend))]
-
+    permission_classes = [IsPublic | (IsOwner | IsAdmin | IsBackend)]
     
     @action(detail=False, permission_classes=[((IsAdmin | IsBackend))])
     def dequeue(self, request):
```

Comment:
some new functionality was added to dequeue since this PR was opened (PRs 97-101 into main). It would be good to update this branch from main. After that, can you revisit this view to make sure everything is in the right place in the try catch statement

Reply:
I tried to create a PR from main to better-errors, but it says there are no changes to review. Can you try doing it? I can check everything is right after it.